docs/community_resources/faq.md

```diff
@@ -0,0 +1,90 @@
+# FAQ
+
+This section contains answers for most common questions, tips around support and maintenance, references to important parts of the documentation and tools useful for developers on their daily basis.
+
+#### What information should I specify when creating a Customer Support ticket?
+
+From the Customer Support perspective the most important thing is version of eZ Platform which is used in the project. The best way to specify it is providing the list of currently installed packages by running composer command:
+``` bash
+composer show ezsystems/*
+```
+
+Besides that all the configuration from `app/config` directory might be helpful. It is good to give steps to reproduce the issue, or at least some meaningful description of circumstances under which the problem occurred.
+
+If you stumbled upon some data inconsistency related problem providing database logs should also be important step.
```

Comment:
Not a big fan of this sentence. Data inconsistency is very serious and very rare - you don't just "stumble upon" it :)
I would not put it in the "Frequently Asked Question" section, as it is not frequent (and putting it in here may suggest otherwise).

Reply:
I changed this sentence a bit, now is more general. What I wanted to say here, is that database problems could also be related to migration. It doesn't necessarily mean that data inconsistencies or corruptions are common ones.

Reply:
:+1: